master/reference/typha/configuration.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Configuring Typha
+canonical_url: 'https://docs.projectcalico.org/v3.1/reference/typha/configuration'
+---
+
+Configuration for Typha is read from one of two possible locations, in
+order, as follows.
+
+1.  Environment variables, prefixed with `TYPHA_`.
+
+2.  The Typha configuration file.  The path to this file defaults to
+    `/etc/calico/typha.cfg` but can be overridden using the `-c` or
+    `--config-file` options on the command line.
+
+The value of any configuration parameter is the value read from the
+*first* location containing a value. For example, if an environment variable
+contains a value, it takes precedence.
```

Comment:
Nice, thanks for clarifying with an example.

Reply:
Thanks, but I can't really claim credit here, as I copied that from the Felix configuration page!